### DISCUSSION TITLE
Antibody Optimization Help - CD45 and Ki67

### DISCUSSION THREAD

**Discussion by LJY1207** [IBEXImagingCommunity/ibex_imaging_knowledge_base#139] [Q&A] [answered]
Hello!

I've been trying to optimize a panel including CD45 and Ki67, and I haven't had great results with either. 

For the CD45, I tried two Novus antibodies (details below), with the one in PE working and the one in AF488 not working despite them being the same clone. More specific details below, and I have also attached images. Any thoughts/feedback/recommendations? I'm searching for a CD45 in either 488, 594, or 647 that works on FFPE. 

Antibody that didn't work:
- Novus Biologicals Cat No. NBP2-34528AF488
- Clone: 2B11 + PD7/26
- Conjugation: AF488
- AR: Leica BOND ER1 30 min + ER2 min
- Dilution: 1:100
- Test tissue: FFPE tonsil 
![Human_Tonsil_FFPE_Hoechst_blue_CD45_AF488_green](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/135713452/f0ecfd80-096d-4311-8a53-8dd48a3d7e16)

Antibody that worked (already in IBEX community database): 
- Novus Biologicals Cat No. NBP2-34528PE
- Clone: 2B11 + PD7/26
- Conjugation: PE
- AR: Leica BOND ER1 30 min + ER2 min
- Dilution: 1:100
- Test tissue: FFPE tonsil 
![Human_Tonsil_FFPE_Hoechst_blue_CD45_PE_green](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/135713452/b111a25a-66fb-40b4-9638-42bfddd59151)

For the Ki67, I've been encountering a strange membranous staining. I am using cystic duct FFPE and have attached an image. Further details below, and this combination has worked beautifully in tonsil. Has anyone encountered this issue before? Any recommendations on alternative antibodies or how to troubleshoot?

- Novus Biologicals Cat No. NB500-170B
- Clone: polyclonal
- Conjugation: Biotin
- Secondary: Streptavidin AF555 (Thermo Fisher Cat. No. S32355)
- AR: Leica BOND ER1 30 min + ER2 min
- Dilution: 1:25 (primary); 1:200 (secondary)
- Test tissue: FFPE cystic duct 

![Human_cystic_duct_FFPE_Hoechst_blue_Ki67_1to25_SA555_1to200_red](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/135713452/635af856-5734-4eba-a233-aff60d7c3154)

Thank you in advance!!

**Comment by radtkea** [comment, marked as answer]
Thank you for very much for your question, experimental details, and GREAT pictures!

**CD45 AF488**
I agree with your conclusion that [CD45 AF488](https://www.novusbio.com/products/cd45-antibody-2b11-pd7-26_nbp2-34528af488) does not work in your tonsil FFPE tissues. Please contribute as a "not recommended" entry. It looks like the only "signal" is autofluorescence from red blood cells (bright round signal associated with the vessel).

**CD45 PE**
This works great! So glad it is working well for you in the tonsil FFPE and you can reproduce our results with this [antibody](https://www.novusbio.com/products/cd45-antibody-2b11-pd7-26_nbp2-34528pe). Please contribute as an "agree" entry.

**Alternatives for CD45**
_Option 1:_ Use the same clones for CD45 from Novus Biologicals in an HRP conjugate, [CD45-HRP](https://www.novusbio.com/products/cd45-antibody-2b11-pd7-26_nbp2-34528h). Amplify with an [anti-HRP secondary](https://www.jacksonimmuno.com/catalog/products/123-545-021). Caveat: Be careful about the order of the antibodies. If you use an anti-goat secondary, it will have to go in an earlier cycle because IBEX does not strip antibodies and the HRP secondary is raised in goats. The advantage with this approach is it uses the same clones that have been validated previously and allows you to amplify your signal in a channel known to have a lot of autofluorescence (AF488).
_Option 2:_ Use this HuBMAP validated clone for FFPE samples following pH 9 antigen retrieval from [Miltenyi Biotec](https://www.miltenyibiotec.com/US-en/products/cd45-antibody-anti-human-5b1.html#conjugate=fitc:size=100-tests-in-200-ul). It's a FITC conjugate and will require longer dye inactivation ([30 minutes of LiBH4](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html)). You could amplify the signal using an [anti-FITC secondary](https://www.thermofisher.com/antibody/product/FITC-Fluorescein-isothiocyanate-Antibody-clone-F4-1-Monoclonal/X1542M) or [anti-mouse IgG2a secondary](https://www.thermofisher.com/antibody/product/Goat-anti-Mouse-IgG2a-Cross-Adsorbed-Secondary-Antibody-Polyclonal/A-21131). It should be bright enough as a direct conjugate but just suggesting some alternatives for amplifying. Same rules apply: Be careful about antibody order and double check that you do not have any goat or mouse IgG2a antibodies in earlier cycles if you use this approach.

**Ki-67**
It's great that you have it working so well in the tonsil. Please share your "agree" contribution. It is critical to use a [biotin/avidin blocking kit](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/NA_NA/0000-0003-4379-8967.html) when using biotin conjugated antibodies and streptavidin reagents. In addition, I make sure my antibodies are diluted in serum free media when adding biotin-conjugated antibodies. My labeling buffer is: 1X PBS, 0.3% Triton-X-100, and 1% Fc block. No BSA. I suspect that the high background you are seeing is due to the tissue itself, cystic duct. Biotin/Streptavidin detection may not be suitable in this [tissue](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2480577/#:~:text=Large%20amounts%20of%20endogenous%20biotin,tissue%20%5B3%2C%204%5D.). 

Alternatives to try include these [HuBMAP validated antibodies](https://avr.hubmapconsortium.org/?q=ki-67) associated with the [OMAP effort](https://communities.springernature.com/posts/high-affinity-team-creates-spatial-biology-resources-for-construction-of-a-human-reference-atlas).
1. [Abcam Ki-67 antibody](https://www.abcam.com/en-kr/products/primary-antibodies/ki67-antibody-sp6-bsa-and-azide-free-ab197547) validated for FFPE pH 6 and 9. You would need to be conjugate to AF555/PE.
2. [Miltenyi Biotec Ki-67](https://www.miltenyibiotec.com/US-en/products/ki-67-antibody-anti-human-readye-lease-real1047.html#conjugate=fitc:size=200-ul) antibody. Validated for FFPE pH 9. They do not have this antibody in PE but you could move your panel around and use the FITC or APC conjugates, potentially.

Thank you again for providing all these details in the open forum! Let me know if you have any questions about contributing resources. You will be named [here](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/the_who.html) and included in our [Zenodo](https://zenodo.org/records/7693279) site which you can add as a dataset to your CV.

**Reply by LJY1207** [reply]
Hi Andrea,

Thank you so much for the detailed response! I've purchased the CD45-HRP along with a couple anti-HRP secondaries, as well as the Miltenyi Biotec Ki-67 in the APC conjugate. Fingers crossed it arrives soon and works! 

I will be sure to contribute my images and staining information with "recommended"/"not recommended" to the IBEX community database! Hoping it can help some others as the community grows.

**Reply by radtkea** [reply]
Awesome, thank you so much! We are very grateful for your knowledge. Fingers crossed with the antibody validation. May your experiments be fruitful!
